src/Console/ConfigurationResolver.php

```diff
@@ -314,13 +314,15 @@ private function computeConfigFiles()
         }
 
         $path = $this->path;
-        if (is_file($path) && $dirName = pathinfo($path, PATHINFO_DIRNAME)) {
-            $configDir = $dirName;
-        } elseif ($this->isStdIn || null === $path) {
+
+        if ($this->isStdIn || 0 === count($path)) {
             $configDir = $this->cwd;
-            // path is directory
+        } elseif (1 < count($path)) {
+            throw new InvalidConfigurationException('For multiple paths config parameter is required.');
```

Comment:
Yes but projects using `sniff` should have a `.sniff.json` instead of `.php_cs`, which is why they don't have a config (but may have multiple paths).

That's why I wrote a "null config file" which returns an empty config: it's just so that I don't get the error *For multiple paths config parameter is required*.

Anyway I don't want to bore you with all that because that's my fault: I'm using php-cs-fixer in a weird way ;)